core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -609,4 +641,9 @@ private void addSampleProfileRecord(Sample sample) throws DaoException {
     private void missingSample(String stableSampleID) {
         throw new NullPointerException("Sample is not found in database (is it missing from clinical data file?): " + stableSampleID);
     }
+
+    private String convertMapToJsonString(Map<String, Map<String, String>> map) throws JsonProcessingException {
+        ObjectMapper mapper = new ObjectMapper();
+        return mapper.writeValueAsString(map);
```

Comment:
This is the normal way to convert to JSON? It is *really* direct.

Reply:
yes